src/streamlink/plugins/ceskatelevize.py

```diff
@@ -145,10 +115,158 @@ def _get_streams(self):
+        validate.transform(_ctcomp_re.findall),
+        validate.transform(lambda vl: [{"video-id": v[0], "ctcomp-data": json.loads(html_unescape(v[1]))} for v in vl])
+    )
+    _playlist_info_schema = validate.Schema({
+        "type": validate.text,
+        "id": validate.any(validate.text, int),
+        "key": validate.text,
+        "date": validate.text,
+        "requestSource": validate.text,
+        "drm": int,
+        validate.optional("canBePlay"): int,
+        validate.optional("assetId"): validate.text,
+        "quality": validate.text,
+        validate.optional("region"): int
+    })
+
+    def __init__(self, session, url, res=None):
+        self.session = session
+        self.url = url
+        self.response = res
+
+    def _get_streams(self):
+        if self.response is None:
+            infos = self.session.http.get(self.url, schema=self._ctcomp_schema)
+        else:
+            infos = self.session.http.json(self.response, schema=self._ctcomp_schema)
+        if not infos:
+            # playlist infos not found
+            raise PluginError('Cannot find playlist infos!')
+
+        vod_prio = len(infos) == 2
+        for info in infos:
+            try:
+                pl = info['ctcomp-data']['source']['playlist'][0]
+            except KeyError:
+                raise PluginError('Cannot find playlist info!')
+
+            pl = self._playlist_info_schema.validate(pl)
+            if vod_prio and pl['type'] != 'VOD':
+                continue
+
+            log.trace('{0!r}'.format(info))
+            if pl['type'] == 'LIVE':
+                data = {
+                    "contentType": "live",
+                    "items": [{
+                        "id": pl["id"],
+                        "assetId": pl["assetId"],
+                        "key": pl["key"],
+                        "playerType": "dash",
+                        "date": pl["date"],
+                        "requestSource": pl["requestSource"],
+                        "drm": pl["drm"],
+                        "quality": pl["quality"],
+                    }]
+                }
+            elif pl['type'] == 'VOD':
+                data = {
+                    "contentType": "vod",
+                    "items": [{
+                        "id": pl["id"],
+                        "key": pl["key"],
+                        "playerType": "dash",
+                        "date": pl["date"],
+                        "requestSource": pl["requestSource"],
+                        "drm": pl["drm"],
+                        "canBePlay": pl["canBePlay"],
+                        "quality": pl["quality"],
+                        "region": pl["region"]
+                    }]
+                }
+
+        headers = {
+            "Content-Type": "application/x-www-form-urlencoded; charset=UTF-8",
+            'Accept': 'application/json, text/javascript, */*; q=0.01',
+            'Accept-Language': 'de,en-US;q=0.7,en;q=0.3',
```

Comment:
seems wrong or not needed

Reply:
This is probably the minimum required header, but I do not know if it is always like this:
`{"Content-Type": "application/x-www-form-urlencoded; charset=UTF-8"}`

If you have nothing to complain about, then I will adapt the code accordingly?

Reply:
use the minimum required header,
using `'Accept-Language': 'de,en-US;` on a **Czech** website as default seems wrong